specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/2017-11-03-preview/hanaonazure.json

```diff
@@ -0,0 +1,422 @@
+        },
+        "HanaInstancesListResult": {
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/HanaInstance"
+                    },
+                    "description": "The list of SAP HANA on Azure instances."
+                },
+                "nextLink": {
+                    "type": "string",
+                    "description": "The URL to get the next set of HANA instances."
+                }
+            },
+            "description": "The response from the List HANA Instances operation."
+        },
+        "HanaInstance": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/HanaInstanceProperties",
+                    "description": "HANA instance properties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "description": "HANA instance info on Azure (ARM properties and HANA properties)"
+        },
+        "HanaInstanceProperties": {
+            "properties": {
+                "hardwareProfile": {
+                    "$ref": "#/definitions/HardwareProfile",
+                    "description": "Specifies the hardware settings for the HANA instance."
+                },
+                "storageProfile": {
+                    "$ref": "#/definitions/StorageProfile",
+                    "description": "Specifies the storage settings for the HANA instance disks."
+                },
+                "osProfile": {
+                    "$ref": "#/definitions/OSProfile",
+                    "description": "Specifies the operating system settings for the HANA instance."
+                },
+                "networkProfile": {
+                    "$ref": "#/definitions/NetworkProfile",
+                    "description": "Specifies the network settings for the HANA instance."
+                },
+                "hanaInstanceId": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Specifies the HANA instance unique ID."
+                }
+            },
+            "description": "Describes the properties of a HANA instance."
+        },
+        "HardwareProfile": {
+            "properties": {
+                "hardwareType": {
+                    "type": "string",
+                    "enum": [
+                        "Cisco_UCS"
+                    ],
+                    "x-ms-enum": {
+                        "name": "HanaHardwareTypeNamesEnum",
+                        "modelAsString": true
+                    },
+                    "description": "Name of the hardware type (vendor and/or thrie product name)"
+                },
+                "hanaInstanceSize": {
```

Comment:
Are these values a known set?

Reply:
Yes, I will find out all the available sizes and make this an enum

Reply:
done